mne/dipole.py

```diff
@@ -553,9 +581,11 @@ def fit_dipole(evoked, cov, bem, trans=None, n_jobs=1, verbose=None):
     data = data[picks]
     ch_names = [info['ch_names'][p] for p in picks]
     proj_op = make_projector(info['projs'], ch_names, info['bads'])[0]
-    out = _fit_dipoles(data, times, src['rr'], guess_fwd_svd, fwd_data,
+    out = _fit_dipoles(min_dist_to_inner_skull, data, times, src['rr'],
+                       guess_fwd_svd, fwd_data,
                        whitener, proj_op, n_jobs)
     dipoles = Dipole(times, out[0], out[1], out[2], out[3], comment)
     residual = out[4]
+    dist_to_inner_skull = out[5]
     logger.info('%d dipoles fitted' % len(dipoles.times))
-    return dipoles, residual
+    return dipoles, residual, dist_to_inner_skull
```

Comment:
you're changing the API of a public function. You cannot do this. If it's just for the test do differently.